projects/OG-Financial/src/main/java/com/opengamma/financial/analytics/isda/credit/CreditCurveDataKey.java

```diff
@@ -67,6 +68,19 @@
    */
   @PropertyDefinition
   private final RestructuringClause _restructuring;
+
+  /**
+   * The Credit Default Swap type. Defaults to Single Name
+   */
+  @PropertyDefinition
```

Comment:
Still you don't want these enum to be null, right? Alternatively, they could be Optional?